### PR TITLE
G131 rest interface

### PR DESCRIPTION
**Pull request by gchqdev03**
Merge example-rest module into develop branch

### CODE REVIEW

File: example-rest/src/main/webapp/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <script src='js/swagger-ui.js' type='text/javascript'></script>
+    <script src='js/highlight.9.1.0.pack.js' type='text/javascript'></script>
+    <script src='js/highlight.9.1.0.pack_extended.js' type='text/javascript'></script>
+    <script src='js/jsoneditor.min.js' type='text/javascript'></script>
+    <script src='js/marked.js' type='text/javascript'></script>
+    <script src='js/swagger-oauth.js' type='text/javascript'></script>
+    <script type='text/javascript' src='js/gaffer.js'></script>
+
+    <script type="text/javascript">
+$(function (){
```
Comment (p013570) [inline example-rest/src/main/webapp/index.html]: I don't think this function is needed

File: example-rest/src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  });
+  </script>
+</head>
+<body class="swagger-section">
+<div id="header">
+    <div class="swagger-ui-wrap">
+        <a id="logo">Gaffer REST API</a>
+
+        <form id="api_selector">
+
```
Comment (p013570) [inline example-rest/src/main/webapp/index.html]: Please could you remove some of the unnecessary white space.

**Comment by p013570** [comment]
There are a few swagger icons that I don't think we are using and could be removed.

File: example-rest/pom.xml [hunk 3]
```diff
@@ -0,0 +1,130 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (p013570) [inline example-rest/pom.xml]: Missing copyright header

File: example-rest/src/main/webapp/css/gaffer.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.swagger-ui-wrap{
+max-width: 95%;
```
Comment (p013570) [inline example-rest/src/main/webapp/css/gaffer.css]: This seems to be being overridden by 'width: 60%' in print.css.

**Comment by gchqdev03** [comment]
On hold pending licence review of javascript dependencies

**Pull request merged**